### PR TITLE
Add jquery-tokeninput@1.7.0 w/ git auto-update

### PR DESCRIPTION
**Pull request by extend1994**
Pull request for issue: #10774 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/loopj/jquery-tokeninput
 * Official website (optional, not the repository): https://loopj.com/jquery-tokeninput/
 * NPM package url (optional): https://www.npmjs.com/package/jquery-tokeninput
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 84
   - Count of stars: 2,379
   - Count of forks: 1,049
 * NPM download stats (optional):
   - Downloads in the last day: 4
   - Downloads in the last week: 26 
   - Downloads in the last month: 172

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by extend1994** [comment]
@cdnjs/intern3 Please help me review this pull request, thank you.
I decided to use npm as the source because it seems more complete one.

**Comment by extend1994** [comment]
@hare1039 @sufuf3 Please take a look at https://github.com/loopj/jquery-tokeninput/blob/v1.7.0/src/jquery.tokeninput.jsjquery.tokeninput.js. it seem that `jquery.tokeninput.js` of v1.7.0 on GitHub is actually v1.6.1, which is weird. Like what I said "I decided to use npm as the source because it seems more complete one." That's why I use npm.

**Comment by sufuf3** [comment]
Maybe we can discuss with the author? What do you think?

**Comment by extend1994** [comment]
Issue: loopj/jquery-tokeninput#858

**Comment by extend1994** [comment]
Accidentally close and reopen it.

**Comment by sufuf3** [comment]
@extend1994 There is no response from the upstream.
And I think we should add the newest version. In this lib, the files should be added from GitHub.
Although the version in `jquery.tokeninput.js` is different from git tag, I still think we can follow our auto-update mechanism.

What do you think?

Thank you.

**Comment by extend1994** [comment]
@sufuf3 I found npm package is not maintained by the author. So I directly use git as the source. As for 
>Please take a look at https://github.com/loopj/jquery-tokeninput/blob/v1.7.0/src/jquery.tokeninput.js. it seem that jquery.tokeninput.js of v1.7.0 on GitHub is actually v1.6.1, which is weird. 

I remember Peter once said we should recognize version by tags. So it is no longer a problem.
Please help me review this PR. Thank you.

**Comment by extend1994** [comment]
ping @hare1039 @qwe19272375

**Comment by extend1994** [comment]
ping @cdnjs/library-reviewer

File: ajax/libs/jquery-tokeninput/package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    "type": "git",
+    "url": "git+https://github.com/loopj/jquery-tokeninput.git"
+  },
+  "keywords": [
+    "jquery",
+    "tokeninput"
+  ],
+  "author": "James Smith <james@loopj.com> (https://loopj.com)",
+  "homepage": "https://loopj.com/jquery-tokeninput/",
+  "license": "MIT",
```
Comment (maruilian11) [inline ajax/libs/jquery-tokeninput/package.json]: https://github.com/loopj/jquery-tokeninput/blob/master/bower.json#L21
hmmm... Looks like it's a dual license, the problem is how to confirm the GPL license version.

**Comment by extend1994** [comment]
@maruilian11 Thanks.
I tried to find the license version author provided but I could not find it. And considering that the author seem to be not maintaining the lib and what the [url](https://opensource.org/licenses/gpl-license.php) in bower.json says
> If you have licensed software you've written under GPL version 2, and you are the original licensor of that software, you may wish to relicense your software under GPL version 3.

Most GPL version 2 software is actually licensed as "GPL version 2 or any later version", or words to similar effect, which is commonly understood as a grant under GPL version 2, section 9 of an "option to follow the terms and conditions . . . of any later version" of the GPL. In this case, even if you are not the original licensor of the GPL version 2 software, you can "follow the terms and conditions" of GPL version 3 instead. This is not commonly done (at least in any explicit way), however.

So I directly add GPL-3.0 license. Please help me review it.

**Comment by PeterDaveHello** [comment]
@maruilian11 @extend1994 @sufuf3 @qwe19272375 don't we need to update `.gitignore` for the tags `jquery-tokeninput-*`?

**Comment by maruilian11** [comment]
Yes, sorry for not checking that.

**Pull request merged**